tdx-base.xml

```diff
@@ -4,9 +4,9 @@
 
   <default sync-j="4"/>
 
-  <project remote="flashbots"       revision="tdx"              name="yocto-scripts"                   path="srcs/yocto-scripts">
-       <copyfile dest="setup"       src="setup"/>
-       <copyfile dest="Makefile"    src="Makefile"/>
+  <project remote="flashbots"       revision="merge-scripts"     name="yocto-manifests"                 path="srcs/yocto-manifests">
```

Comment:
This will change to revision main instead later. But for testing purposes, I set it up to this branch to check it out first